### DISCUSSION TITLE
Is it $wire.entangle() or $wire.$entangle()?

### DISCUSSION THREAD

**Discussion by nikspyratos** [livewire/livewire#8581] [1. Help] [answered]
Heyo. I've been doing a V2 -> V3 upgrade on a repo making heavy use of `entangle`.

The auto-upgrader output example (and what it replaces) is:
```
+------------------------------+
| Before/After example         |
+------------------------------+
| - $wire.entangle(...)        |
| + $wire.$entangle(..., true) |
+------------------------------+
```

But from the [docs](https://livewire.laravel.com/docs/alpine#sharing-state-using-wireentangle) and my IDE complaining it can't find `$entangle`, I assume it's supposed to be `entangle` instead?

Just want to confirm if this is a bug or not, because it should be a very quick PR to fix.

**Comment by joshhanley** [comment, marked as answer]
@nikspyratos both will work
